src/Chapter4.hs

```diff
@@ -611,6 +628,13 @@ Implement the 'Monad' instance for our lists.
   maybe a few) to flatten lists of lists to a single list.
 -}
 
+instance Monad List where
+  (>>=) :: List a -> (a -> List b) -> List b
+  l >>= f = concatList(fmap f l)
```

Comment:
Great!
Nothe, that in Haskell, function are separated from arguments with spaces 🙂 :
```suggestion
  l >>= f = concatList (fmap f l)
```